<?xml version="1.0" encoding="UTF-8"?>
<!-- Path: gui\informacao_produto\version_edit_dialog.ui -->
<ui version="4.0">
 <class>VersionEditDialog</class>
 <widget class="QDialog" name="VersionEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar Versão</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Editar Versão</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>530</width>
        <height>579</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="idLabel">
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="idLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="uuidLabel">
         <property name="text">
          <string>UUID:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="uuidLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="versaoLabel">
         <property name="text">
          <string>Versão:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="versaoLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="nomeLabel">
         <property name="text">
          <string>Nome:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="nomeLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="tipoVersaoLabel">
         <property name="text">
          <string>Tipo de Versão:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="tipoVersaoComboBox"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="subtipoLabel">
         <property name="text">
          <string>Subtipo de Produto:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="subtipoComboBox"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="loteLabel">
         <property name="text">
          <string>Lote:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="loteComboBox"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="orgaoProdutorLabel">
         <property name="text">
          <string>Órgão Produtor:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="orgaoProdutorLineEdit"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="palavrasChaveLabel">
         <property name="text">
          <string>Palavras-chave:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="palavrasChaveLineEdit">
         <property name="placeholderText">
          <string>Separadas por vírgula</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="dataCriacaoLabel">
         <property name="text">
          <string>Data de Criação:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QDateEdit" name="dataCriacaoDateEdit">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="dataEdicaoLabel">
         <property name="text">
          <string>Data de Edição:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QDateEdit" name="dataEdicaoDateEdit">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="descricaoLabel">
         <property name="text">
          <string>Descrição:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QTextEdit" name="descricaoTextEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="metadadoLabel">
         <property name="text">
          <string>Metadados:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QTextEdit" name="metadadoTextEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Monospace</family>
          </font>
         </property>
         <property name="placeholderText">
          <string>Metadados em formato JSON</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VersionEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VersionEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>